assets/javascript/game.js

```diff
@@ -25,11 +25,21 @@ window.Game = React.createClass({
       }
 
       $.get('/game/' + this.props.gameID, (data) => {
+          if (this.state.game && data.round != this.state.game.round) {
+            this.changedTurn();
+          }
           this.setState({game: data});
           setTimeout(this.refresh, 3000);
       });
     },
 
+    changedTurn: function() {
+        $('#clue-word')[0].value = "";
+        $('#clue-word')[0].disabled = false;
+        $('#clue-count')[0].value = "";
+        $('#clue-count')[0].disabled = false;
```

Comment:
I think in "The React Way" these should be a part of the state and then incorporated in render.